thanos/charts/templates/tests/test-thanos-config.yaml

```diff
@@ -17,6 +17,9 @@ data:
 # SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
 # SPDX-License-Identifier: Apache-2.0
 
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This thing is borked somehow.

```suggestion
```

Reply:
https://github.com/cloudoperators/greenhouse-extensions/pull/324